### DISCUSSION TITLE
Personal access token (classic) Bug

### DISCUSSION THREAD

**Discussion by BonanseaMariano** [community/community#129211] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

i cannot select both repo and workflow access together, its either one or the other. This has broke the JetBrains IDEs integration.

**Comment by 21shahvage** [comment]
any workarounds?

**Reply by vatsalcshah** [reply]
Workaround:

As a temporary solution, you can create two separate personal access tokens (classic):

Token 1: Grant this token access to repositories.
Token 2: Grant this token access to workflows.
Use the appropriate token depending on the specific functionality you need in your JetBrains IDE.
